app/pages/MyGamesPage/index.tsx

```diff
@@ -1,20 +1,121 @@
-import React from 'react';
-import { Grid, PageLoading } from 'store-library';
-import { Container } from 'store-library/src/styles';
+import React, { useState } from 'react';
+import { Grid, PageLoading, Dropdown } from 'store-library';
+import {
+  Container,
+  Caps11,
+  WHITE_05,
+  WHITE,
+  Caption13,
+} from 'store-library/src/styles';
 import MyGameCard from 'components/MyGameCard';
 import { useAppsState } from 'containers/Apps';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import { AppState } from 'proto/stoold_pb';
+import { compareGamesByLastPlayed, compareGamesByTitle } from 'helpers';
 
 const { Row, Col } = Grid;
 
+const filterOptions = [
+  {
+    title: 'filter.all',
+    value: 'all',
+  },
+  {
+    title: 'filter.installed',
+    value: 'installed',
+  },
+];
+
+const sortOptions = [
+  {
+    title: 'sort_options.lastPlayed',
+    value: 'lastPlayed',
+  },
+  {
+    title: 'sort_options.alphabet',
+    value: 'alphabet',
+  },
+];
+
 const MyGamesPage = () => {
-  const { apps, loading, error } = useAppsState();
+  const { apps: appsData, loading, error } = useAppsState();
+  const [filter, setFilter] = useState('all');
+  const [sort, setSort] = useState('lastPlayed');
+  const { t } = useTranslation();
 
   if (loading) return <PageLoading />;
 
   if (error) return <div>{error.message}</div>;
 
+  const filterTitle = (
+    <Title>
+      {t('labels.filter')}
+      :&nbsp;
+      <White>{t(`filter.${filter}`)}</White>
+    </Title>
+  );
+
+  const sortTitle = (
+    <Title>
+      {t('labels.sorting')}
+      :&nbsp;
+      <White>{t(`sort_options.${sort}`)}</White>
+    </Title>
+  );
+
+  const onChangeFilter = (value: string) => {
+    setFilter(value);
+  };
+
+  const onChangeSort = (value: string) => {
+    setSort(value);
+  };
+
+  const apps = appsData
+    .filter(app => {
+      if (filter === 'installed') {
+        return app.state !== AppState.APPSTATE_NOT_INSTALLED;
+      }
+
+      return true;
+    })
+    .sort((a, b) =>
```

Comment:
```suggestion
    .sort(sort === 'lastPlayed' ? compareGamesByLastPlayed : compareGamesByTitle)
```
Может лучше так?
Так только один раз проверка будет, а не на каждой итерации.

Reply:
Ну да)